packages/neos-ui-guest-frame/src/InlineUI/NodeToolbar/index.js

```diff
@@ -164,7 +164,7 @@ export default class NodeToolbar extends PureComponent {
         });
 
         return (
-            <div className={classNames} style={toolbarPosition}>
+            <div className={classNames} data-ignore_click_outside="true" style={toolbarPosition}>
```

Comment:
add a comment above the return statement explaining the data-ignore_click_outside is a speciality for the enhanceWithClickOutside